examples/rocket_example/Cargo.toml

```diff
@@ -23,7 +23,7 @@ serde_json = { version = "^1" }
 [dependencies.sea-orm]
 path = "../../" # remove this line in your own project
 version = "^0.5.0"
-features = ["macros", "runtime-tokio-native-tls"]
+features = ["macros", "mock", "runtime-tokio-native-tls"]
```

Comment:
We dont actually have to add `mock` here right?

Reply:
Not needed. I will revert it

Reply:
Reverted